src/pages/RealEstateDetail/Deal/DealWrap/DealDetail.js

```diff
@@ -3,10 +3,13 @@ import KakaoMapScript from '../../KakaoMapScript';
 
 import * as S from '../DealStyle';
 
-function DealDetail() {
+function DealDetail(props) {
```

Comment:
매개변수 자리에서 바로 구조분해 가능합니다.
```suggestion
function DealDetail({mortagageInfoLatitue, mortagageInfoLongitude, ~~~ }) {
```